### PR TITLE
feat: disable paste on firefox

### PR DESCRIPTION
**Pull request by PatriciaHeimfarth**
### What kind of change does this PR introduce?
https://github.com/phcode-dev/phoenix/issues/60

### Screenshots or Gifs of the change
grafik
grafik

### CODE REVIEW

**Comment by phoenixide** [comment]
### Error: Contributor Licence Agreement Signature Missing
Please sign the Contributor Licence Agreement by clicking the following link.
<p align="center"> <a href="https://phcode-dev.github.io/contributor-license-agreement/">Click here to sign the CLA</a></p>
### Error: Contributor Licence Agreement Signature Missing
The following commiter(s) has not signed the Contributor Licence Agreement:
PatriciaHeimfarth
Please sign the Contributor Licence Agreement by clicking the following link. 
<p align="center"> <a href="https://phcode-dev.github.io/contributor-license-agreement/">Click here to sign the CLA</a></p>

**Comment by abose** [comment]
@PatriciaHeimfarth Many thanks for contributing to phcode.dev :) . I would like to suggest some changes:
1. Disabling the command entirely in firefox will have unforeseen problems if extensions programmatically trigger the paste command. Search for `addMenuItem(Commands.EDIT_PASTE)` in the codebase and put that under the same firefox check.
2. Platform checks should use the global `Phoenix.browser ` object. It is required as Phoenix loads in a firefox webview in `Tauri` in Linux. And we are adding a few more platforms under ios/android in near future. Tauri brings in paste support. So target the fix to `!Phoenix.browser.isTauri && Phoenix.browser.desktop?.isFirefox`
image

**Comment by PatriciaHeimfarth** [comment]
I updated the pr, is it okay like this?

**Comment by phoenixide** [comment]
### Error: Contributor Licence Agreement Signature Missing
Please sign the Contributor Licence Agreement by clicking the following link.
<p align="center"> <a href="https://phcode-dev.github.io/contributor-license-agreement/">Click here to sign the CLA</a></p>
### Error: Contributor Licence Agreement Signature Missing
The following commiter(s) has not signed the Contributor Licence Agreement:
PatriciaHeimfarth
Please sign the Contributor Licence Agreement by clicking the following link. 
<p align="center"> <a href="https://phcode-dev.github.io/contributor-license-agreement/">Click here to sign the CLA</a></p>

File: src/command/DefaultMenus.js [hunk 1]
```diff
@@ -277,8 +277,9 @@ define(function (require, exports, module) {
         editor_cmenu.addMenuDivider();
         editor_cmenu.addMenuItem(Commands.EDIT_CUT);
         editor_cmenu.addMenuItem(Commands.EDIT_COPY);
-        editor_cmenu.addMenuItem(Commands.EDIT_PASTE);
-
+        if(!(!window.Phoenix.browser.isTauri && window.Phoenix.browser.desktop.isFirefox)){
```
Comment (abose) [inline src/command/DefaultMenus.js:280]: `!window.Phoenix.browser.desktop.isFirefox || window.Phoenix.browser.isTauri` might be clearer to read.

Comment (abose) [reply src/command/DefaultMenus.js:280]: There should be one more place where the paste menu is there in the editor context menu: 
image

**Comment by PatriciaHeimfarth** [comment]
Hi, I did the update, can you have another look?

**Comment by phoenixide** [comment]
### Error: Contributor Licence Agreement Signature Missing
Please sign the Contributor Licence Agreement by clicking the following link.
<p align="center"> <a href="https://phcode-dev.github.io/contributor-license-agreement/">Click here to sign the CLA</a></p>
### Error: Contributor Licence Agreement Signature Missing
The following commiter(s) has not signed the Contributor Licence Agreement:
PatriciaHeimfarth
Please sign the Contributor Licence Agreement by clicking the following link. 
<p align="center"> <a href="https://phcode-dev.github.io/contributor-license-agreement/">Click here to sign the CLA</a></p>

**Comment by abose** [comment]
@PatriciaHeimfarth Thanks for the changes. LGTM for merging.

Please sign the contributor License agreement with phcode.dev : https://phcode-dev.github.io/contributor-license-agreement/ 

It is required so that you as the author, grant phcode.dev permission to distribute your code under AGPL license.

**Comment by PatriciaHeimfarth** [comment]
I signed it via PR, is it correct like this?

**Comment by phoenixide** [comment]
## Thank You for making this pull request.

**Comment by abose** [comment]
LGTM . Merging.
@PatriciaHeimfarth Thanks for contributing to phcode. The changes will reach prod in a week or sooner.

In the meantime, please see [this link](https://github.com/phcode-dev/phoenix#previewing-changes-in-dev-and-staging) on how to preview the changes in the dev stack.

The GitHub issues tab has a lot of feature requests/bugs if you would like to contribute more. Any help would be much appreciated!

**Pull request merged**